operator/pkg/ciliumidentity/cache.go

```diff
@@ -52,7 +52,7 @@ func (c *CIDState) Upsert(id string, k *key.GlobalIdentity) {
 		return
 	}
 
-	c.logger.Debug("Upsert internal mapping between", logfields.CIDName, id, "labels", k.Labels().String())
+	c.logger.Info("Upsert internal mapping", logfields.CIDName, id)
```

Comment:
Does this log message make sense to a user?

If we're going to make these Info level then the user should be able to interpret what they mean so they can get a sense for how to interact with them. I don't think that these logs currently satisfy that expectation.

Reply:
I'm not really sure CIDState transformations should be raised to Info level if this PR already prints allocations in the other change in the reconciler below.

Reply:
I reverted it to Debug, but still I removed the labels, because as mentioned the labels are logged in a separate log anyway.

Reply:
Being a debug only log, IMO it makes sense to keep the labels in the log message to help debugging.